cmd/execution/main.go

```diff
@@ -123,6 +129,7 @@ func main() {
 			flags.StringVarP(&rpcConf.ListenAddr, "rpc-addr", "i", "localhost:9000", "the address the gRPC server listens on")
 			flags.BoolVar(&rpcConf.RpcMetricsEnabled, "rpc-metrics-enabled", false, "whether to enable the rpc metrics")
 			flags.StringVar(&triedir, "triedir", datadir, "directory to store the execution State")
+			flags.StringVar(&executionDataDir, "execution-data-dir", filepath.Join(homedir, ".flow", "execution_data"), "directory to use for Execution Data blobstore")
```

Comment:
Somewhat concerned this might get confusing, since we already have a `/execution` folder that holds the mtries. Although maybe `execution` and `execution_data` is sufficiently distinct?